pages/friendlist.js

```diff
@@ -0,0 +1,283 @@
+        <StyledLogoWrapper>
+          <Logo />
+        </StyledLogoWrapper>
+        <StyledAppName> MeetMate</StyledAppName>
+      </StyledHeadlineBox>
+      <StyledHeadline>Your Mates</StyledHeadline>
+      <StyledCardSection>
+        <StyledFriendCard>
+          <StyledDivLeft>
+            <StyledActivityName>Maximilian</StyledActivityName>
+          </StyledDivLeft>
+          <StyledDivRight>
+            <StyledSection></StyledSection>
+          </StyledDivRight>
+        </StyledFriendCard>
+        <StyledFriendCard>
+          <StyledDivLeft>
+            <StyledActivityName>Annabelschnell</StyledActivityName>
+          </StyledDivLeft>
+          <StyledDivRight>
+            <StyledSection></StyledSection>
+          </StyledDivRight>
+        </StyledFriendCard>
+        <StyledFriendCard>
+          <StyledDivLeft>
+            <StyledActivityName>Peter Enis</StyledActivityName>
+          </StyledDivLeft>
+          <StyledDivRight>
+            <StyledSection></StyledSection>
+          </StyledDivRight>
+        </StyledFriendCard>
+        <StyledFriendCard>
+          <StyledDivLeft>
+            <StyledActivityName>Peli kann</StyledActivityName>
+          </StyledDivLeft>
+          <StyledDivRight>
+            <StyledSection></StyledSection>
+          </StyledDivRight>
+        </StyledFriendCard>
+        <StyledFriendCard>
+          <StyledDivLeft>
+            <StyledActivityName>Hom Tanks</StyledActivityName>
+          </StyledDivLeft>
+          <StyledDivRight>
+            <StyledSection></StyledSection>
+          </StyledDivRight>
+        </StyledFriendCard>
+        <StyledFriendCard>
+          <StyledDivLeft>
+            <StyledActivityName>Maximilian</StyledActivityName>
+          </StyledDivLeft>
+          <StyledDivRight>
+            <StyledSection></StyledSection>
+          </StyledDivRight>
+        </StyledFriendCard>
+        <StyledFriendCard>
+          <StyledDivLeft>
+            <StyledActivityName>Annabelschnell</StyledActivityName>
+          </StyledDivLeft>
+          <StyledDivRight>
+            <StyledSection></StyledSection>
+          </StyledDivRight>
+        </StyledFriendCard>
+        <StyledFriendCard>
+          <StyledDivLeft>
+            <StyledActivityName>Peter Enis</StyledActivityName>
+          </StyledDivLeft>
+          <StyledDivRight>
+            <StyledSection></StyledSection>
+          </StyledDivRight>
+        </StyledFriendCard>
+        <StyledFriendCard>
+          <StyledDivLeft>
+            <StyledActivityName>Peli kann</StyledActivityName>
+          </StyledDivLeft>
+          <StyledDivRight>
+            <StyledSection></StyledSection>
+          </StyledDivRight>
+        </StyledFriendCard>
+        <StyledFriendCard>
+          <StyledDivLeft>
+            <StyledActivityName>Hom Tanks</StyledActivityName>
+          </StyledDivLeft>
+          <StyledDivRight>
+            <StyledSection></StyledSection>
+          </StyledDivRight>
```

Comment:
This seems like way to many unneeded elements. Please try to remove unnecessary ones.
Secondly, please map over a "friends" array to generate all elements instead of hard coding every single one. 
I think you should be able to combine StyledFriendCard,StyledDivLeft and StyledActivityName into one styled component, depending on what you try to achieve with this setup.

Reply:
We don't have any data right now, this is going to be fixed in one of the upcoming stories but still we need to be able to check all the functions like scrolling behavior an the look's